### DISCUSSION TITLE
New release: 2.2.0

### DISCUSSION THREAD

**Discussion by colinodell** [thephpleague/commonmark#792] [Announcements]
### Added

 - Added new `ConverterInterface`
 - Added new `MarkdownToXmlConverter` class
 - Added new `HtmlDecorator` class which can wrap existing renderers with additional HTML tags
 - Added new `table/wrap` config to apply an optional wrapping/container element around a table (#780)

### Changed

 - `HtmlElement` contents can now consist of any `Stringable`, not just `HtmlElement` and `string`

### Deprecated

 - Deprecated `MarkdownConverterInterface` and its `convertToHtml()` method; use `ConverterInterface` and `convert()` instead

<hr /><em>This discussion was created from the release <a href='https://github.com/thephpleague/commonmark/releases/tag/2.2.0'>2.2.0</a>.</em>

**Comment by m6w6** [comment]
Maybe I'm doing something wrong, but isn't this deprecation always triggered, because `MarkdownConverter` still loads `MarkdownConverterInterface`?

**Reply by colinodell** [reply]
Yeah, you're right, that's going to be too noisy since there's really no way to avoid it.  I'll remove the trigger from that particular interface in 2.2.1, to be released in a few minutes.  Thanks for pointing this out!
